apps/src/templates/manageStudents/PasswordReset.jsx

```diff
@@ -0,0 +1,78 @@
+import React, {Component, PropTypes} from 'react';
+import Button from '../Button';
+import i18n from "@cdo/locale";
+
+const styles = {
+  input: {
+    width: 100,
+    height: 29,
+    marginTop: '-25px',
+    marginRight: 10,
+  }
+};
+
+class PasswordReset extends Component {
+  static propTypes = {
+    resetAction: PropTypes.func.isRequired,
+    isResetting: PropTypes.bool
+  };
+
+  state = {
+    isResetting: !!this.props.isResetting,
```

Comment:
Have you validated that this works? I'm not 100% that `this.props` will be set yet at the time when this is evaluated.

Reply:
I verified this works 👍